config/examples/Creality/CR-10 V3/Configuration.h

```diff
@@ -1452,8 +1452,8 @@
   #define FIL_RUNOUT_PIN 2 // Creality CR10-S stock sensor
 
   #define FIL_RUNOUT_STATE     LOW        // Pin state indicating that filament is NOT present.
-  #define FIL_RUNOUT_PULLUP               // Use internal pullup for filament runout pins.
-  //#define FIL_RUNOUT_PULLDOWN           // Use internal pulldown for filament runout pins.
+  //#define FIL_RUNOUT_PULLUP               // Use internal pullup for filament runout pins.
+  #define FIL_RUNOUT_PULLDOWN           // Use internal pulldown for filament runout pins.
```

Comment:
For me it makes more sense to have a pulldown here as the RUNOUT state is LOW.
But neither with pullup or pulldown i'm not able to get a runout condition by unplugging the sensor during a print.
If you try with real filament in the sensor, in and out, it works fine.

Reply:
If disconnecting the switch does nothing then the switch is likely a Normally Open (NO) switch. Filament runout is triggered only when the state _changes_ from HIGH to LOW.